content/en/docs/tasks/policy-enforcement/rate-limit/index.md

```diff
@@ -196,10 +196,11 @@ spec:
       app: productpage
   configPatches:
     - applyTo: HTTP_FILTER
-      listener:
-        filterChain:
-          filter:
-            name: "envoy.filters.network.http_connection_manager"
+      match:
```

Comment:
Thanks for catching this. I think we can just get rid of match here since apply to http_filter implies http_connection_manager. That is what we do at istio integration testing.